### PR TITLE
docs(plugin-text-classification): Plugin Documentation #BOT-372

### PR DESCRIPTION
**Pull request by elozano98**
## Description
Documentation of `plugin-text-classification`.

### CODE REVIEW

File: packages/botonic-plugin-text-classification/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```bash
+npm install @botonic/plugin-text-classification
+```
+
+> **Note**: Windows users should first use the command
+> `npm install --global --production windows-build-tools --vs2015`
+
+## Require the plugin
+
+The plugin must be required in the `src/plugins.js` and defining the locales of trained models into the options.
```
Comment (dpinol) [inline packages/botonic-plugin-text-classification/README.md:22]: "and defining the locales of trained models into the options." -> and the locales of the trained models must be defined in their options

File: packages/botonic-plugin-text-classification/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+> **Note:** The order of provided locales matters. In case locale detection fails, the first one in the array will be used as a fallback.
```
Comment (dpinol) [inline packages/botonic-plugin-text-classification/README.md:24]: the provided locales

**Review by dpinol — CHANGES_REQUESTED** [review]
add the "document" label to the PR, and talk to Anne on what she needs for https://botonic.io/docs.
Talk to Marc on how to create a sample bot for our samples site

File: packages/botonic-plugin-text-classification/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Botonic Plugin Text Classification
+
+## What does this plugin do?
+
+Botonic Plugin Text Classification uses the trained models defined by the user to classify the input text.
+
+After detecting the locale of the text, and using the corresponding model to predict the class, extra information about the intent of the user is added to the input message.
```
Comment (annehubtype) [inline packages/botonic-plugin-text-classification/README.md:7]: After detecting the text locale, and using the relevant model to predict the class, extra information about the user intent is added to the input message.

File: packages/botonic-plugin-text-classification/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Botonic Plugin Text Classification
+
+## What does this plugin do?
+
+Botonic Plugin Text Classification uses the trained models defined by the user to classify the input text.
+
+After detecting the text locale, and using the corresponding model to predict the class, extra information about the user's intent is added to the input message.
```
Comment (vanbasten17) [inline packages/botonic-plugin-text-classification/README.md:7]: `is added to the input object`

File: packages/botonic-plugin-text-classification/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Install the plugin
```
Comment (vanbasten17) [inline packages/botonic-plugin-text-classification/README.md:9]: Installing the plugin

File: packages/botonic-plugin-text-classification/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+From your project, install the plugin by using the following command:
+
+```bash
+npm install @botonic/plugin-text-classification
+```
+
+> **Note**: Windows users should first use the command
+> `npm install --global --production windows-build-tools --vs2015`
+
+## Require the plugin
```
Comment (vanbasten17) [inline packages/botonic-plugin-text-classification/README.md:20]: Requiring the plugin (for the docs I've seen so far, it's common to use this tense)

File: packages/botonic-plugin-text-classification/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+The plugin must be required in the `src/plugins.js` and the locales of the trained models must be defined in their options.
```
Comment (vanbasten17) [inline packages/botonic-plugin-text-classification/README.md:22]: I would say just in `src/plugins.js` or `in the src/plugins.js file`

File: packages/botonic-plugin-text-classification/README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+**Locales order matters.** In case locale detection fails, the first one in the array will be used as a fallback.
```
Comment (vanbasten17) [inline packages/botonic-plugin-text-classification/README.md:24]: Talk in "positive" terms (for developer's confidence) we should not state that XXX can fail, try to rephrase it like, or similar:
> **Note:** If locale detection is ambiguous, the  locale defined first in the array will be used as a fallback.
:)

**Review by vanbasten17 — APPROVED** [review]

**Review by annehubtype — APPROVED** [review]

File: packages/botonic-plugin-text-classification/README.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    options: {
+      locales: ['en', 'es'],
+    },
+  },
+]
+```
+
+## Use
+
+You can use the information added by this plugin to create new routes in **`src/routes.js`**.
```
Comment (annehubtype) [inline packages/botonic-plugin-text-classification/README.md:42]: Use the information added by this plugin to create new routes in **`src/routes.js`** and start using text classification.

**Pull request merged**